packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -618,19 +619,23 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Pro
     eventType.schedulingType === SchedulingType.ROUND_ROBIN ||
     allUsersAvailability.length > 1;
 
+  const seatsMinimumBookingNoticeActive = !!(
+    eventType.seatsPerTimeSlot && eventType.seatsMinimumBookingNotice
+  );
   const timeSlots = getSlots({
     inviteeDate: startTime,
     eventLength: input.duration || eventType.length,
     offsetStart: eventType.offsetStart,
     dateRanges: aggregatedAvailability,
-    minimumBookingNotice: eventType.minimumBookingNotice,
+    minimumBookingNotice: seatsMinimumBookingNoticeActive ? 0 : eventType.minimumBookingNotice,
```

Comment:
we pass in minimumBookingNotice as 0, if our seatsMBN flag is active, so we can handle the out of bounds filtering later on.